### ISSUE TITLE
Error 500 when trying to register my YubiKey on the demo

### ISSUE THREAD

**Issue by metamatik** [Bouke/django-two-factor-auth#367]
I'm currently evaluating solutions to improve the security of my current Django project.
The possibility of providing different 2FA mechanisms out of the box is appealing.
However, upon pressing my YubiKey to input an OTP, the demo site triggers a server error.

## Expected Behavior
Well, I'm expecting... some kind of registration confirmation page, I guess? :)

## Current Behavior
Upon pressing my YubiKey's input button, the input field gets populated with a long string, as expected.
Then the form gets auto-submitted, which in turns triggers a server error (500).

## Steps to Reproduce (for bugs)
1. Log into the demo website with a registered account
2. Click the _Enable Two-Factor Authentication_ button
3. Click the _Next_ button
4. Select the _YubiKey_ radio button, then click the _Next_ button
5. Click into the _YubiKey_ input field, then plug a YubiKey into a USB port and put your finger on its activation zone.
6. The input field gets populated with a long string
7. The form gets auto-submitted
8. The browser shows a _Server Error (500)_ message

## Your Environment
* Browser and version: Latest versions of Chrome and Firefox on macOS Catalina 10.15.5

**Comment by moggers87** [comment]
Are you able to run the example app locally and paste in the exception here? I don't have access to the example app (Heroku seems to have forgotten that I had access, or I'm logged in with the wrong account) and I don't have a Yubikey to test with.

**Comment by metamatik** [comment]
Thanks for getting back to me so quickly :)

Work day's over here in Paris, I'll check that tomorrow and keep you posted!

**Comment by metamatik** [comment]
Hi Matt!

I've been able to reproduce the bug locally, and then empirically "fix it" thanks to [this very recent StackOverflow post](https://stackoverflow.com/questions/62997273/django-and-yubikey-integrate).

Tl;dr: you might want to check if _Use SSL_ is checked in the relevant _YubiKey validation service_ admin form ^^

**Comment by moggers87** [comment]
Thanks for the info. The validation service is part of django-otp-yubikey and it seems they default to not using SSL. I'll open a bug report there and see if they're willing to change that. I doubt you're the first person to run into problems with this.

For what it's worth, our documentation does say to set `use_ssl` to `True` when creating a the validation service: https://django-two-factor-auth.readthedocs.io/en/stable/installation.html#yubikey-setup

**Comment by moggers87** [comment]
Actually, having re-read that, it only specifies `use_ssl=True` when using the shell. The docs could be a bit clearer for those who use the admin site.

**Comment by moggers87** [comment]
With the release of django-otp-yubikey 1.0.0, so I think this can be closed now.

See django-otp/django-otp-yubikey#3
